Source/loadsave.cpp

```diff
@@ -468,6 +468,11 @@ void LoadPlayer(LoadHelper &file, Player &player)
 	player._pBFrames = file.NextLE<int32_t>();
 	file.Skip(4); // skip _pBWidth
 
+	if (player._pmode == PM_SPELL)
+		player.AnimInfo.triggerFrame = player._pSFNum - 1;
+	if (player._pmode == PM_ATTACK || player._pmode == PM_RATTACK)
+		player.AnimInfo.triggerFrame = player._pAFNum - 1;
+

```

Comment:
I was wondering where to place it.
Alternative would be some init player or similiar.